src/f32/vec3.rs

```diff
@@ -790,6 +790,12 @@ impl Vec3 {
     pub fn abs_diff_eq(self, other: Self, max_abs_diff: f32) -> bool {
         abs_diff_eq!(self, other, max_abs_diff)
     }
+
+    /// Returns the angle between two vectors, in radians.
+    #[inline]
+    pub fn angle(self, other: Self) -> f32 {
+        f32::atan2(self.cross(other).length(), self.dot(other))
```

Comment:
I've usually seen this done with acos. Do you know how the atan2 version compares performance wise?

Reply:
acos has issues when the vectors are approaching parallel or opposite:
https://www.jwwalker.com/pages/angle-between-vectors.html

This is also the implementation in cgmath uses.